components/elasticsearch/scripts/upgrade_validation.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+
+import urllib2
+import urlparse
+from os.path import join, dirname
+
+from cloudify import ctx
+
+ctx.download_resource(
+    join('components', 'utils.py'),
+    join(dirname(__file__), 'utils.py'))
+
+import utils  # NOQA
+
+ES_SERVICE_NAME = 'elasticsearch'
+
+install_properties = utils.ctx_factory.get_install_properties(
+    ES_SERVICE_NAME)
+upgrade_properties = utils.ctx_factory._load_ctx_properties(
+    ES_SERVICE_NAME)
+
```

Comment:
This would change with 'get()' after merging https://github.com/cloudify-cosmo/cloudify-manager-blueprints/pull/291/files#diff-23cb882db3b8314e49ad99212e1c4118R738